doc/ref/cli/salt-ssh.rst

```diff
@@ -19,13 +19,17 @@ Salt SSH allows for salt routines to be executed using only SSH for transport
 Options
 =======
 
-.. option:: -r, --raw, --raw-shell
+.. program:: salt
```

Comment:
`salt` --> `salt-ssh`

Reply:
Didn't even notice that, I only moved that line around.